db/migrate/20140902102858_convert_ipam_to_string.rb

```diff
@@ -1,30 +1,14 @@
 class ConvertIpamToString < ActiveRecord::Migration[4.2]
-  class FakeSubnet < ApplicationRecord
-    self.table_name = 'subnets'
-  end
-
   def up
     add_column :subnets, :ipam_tmp, :string, :default => IPAM::MODES[:dhcp], :null => false, :limit => 255
```

Comment:
If you're dropping the migration, there's no reason for a tmp column. Can't this be squashed down to a `change_column`? Same comment for `down`.